shared/locales/langpack/desc.en-US.properties

```diff
@@ -0,0 +1,34 @@
+# This file provides localization strings for the Language Pack.
+
+###
+# The name of the language pack and its description will be displayed
+# to the user when browsing the list of language packs on the marketplace
+# and during installation.
+#
+# IMPORTANT: these strings need to be adapted to your language, not simply 
+# translated. "Locale" needs to be replaced with the localized version 
+# of your language name: "Italiano" for Italian, "Polski" for Polish, etc. 
+# 
+# If the region is relevant for your locale, add it (e.g. "English (US)").
+# Otherwise drop the " (US)".
+# 
+#
+# brandShortName will be taken from shared/locales/branding/
+###
+langpack.name = English (US) Language Pack
+langpack.description = A package providing an American English localization of {{ brandShortName }}.
+
+###
+# Those two strings will be used to display the name and description
+# of the language pack with your language to the user not using your language.
+# It should be an English translation of the localized name/description of
+# your language pack.
+###
+langpack_enUS.name = English (US) Language Pack
+langpack_enUS.description = A package providing an American English localization of {{ brandShortName }}.
```

Comment:
Could we name the keys:

    langpack.native.name
    langpack.native.description
    langpack.english.name
    langpack.english.description

?